app/src/main/res/values/strings.xml

```diff
@@ -310,4 +314,20 @@
     <string name="content_description_ping_everyone">Ping everyone</string>
     <string name="label_message_sent_failure">Message could not be sent.</string>
     <string name="label_try_again">Try again</string>
+
+    <!--sso dialog-->
+    <string name="sso_erro_dialog_title">Unable to log in with SSO</string>
+    <string name="sso_erro_dialog_message">Please try again. If this does not help, contact your admin and provide this code:\nTitle: %s\nCode: %d"</string>
```

Comment:
On designs there is only place for the code, we should omit the title, it's not very meaningful anyway and user won't do anything with that